src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -53,11 +56,12 @@ class Publisher(private val topicName: String) {
         ApiFutures.addCallback(future, object : ApiFutureCallback<String> {
 
             override fun onFailure(throwable: Throwable) {
-                println(throwable)
+                logger.info(throwable.toString())
```

Comment:
You'll normally want error and log not only the message but also the stack trace. In slf4j API if Throwable comes as the last argument of any log methods, it prints the stack trace ([docs](https://www.slf4j.org/faq.html#paramException))

Reply:
Done